backend/src/services/service_factory.py

```
@@ -334,6 +339,46 @@ async def _create_core_services(self) -> None:
+            logger.error(f"Failed to create optimized guide service: {e}")
+            raise ConfigurationError(f"Optimized guide service creation failed: {e}")
+    
+    def create_luxury_guide_service(self) -> LuxuryGuideService:
+        """Create luxury guide service"""
+        try:
+            return LuxuryGuideService()
+        except Exception as e:
+            logger.error(f"Failed to create luxury guide service: {e}")
+            raise ConfigurationError(f"Luxury guide service creation failed: {e}")
+    
+    def create_magazine_pdf_service(self, destination: str = None) -> MagazinePDFService:
```

Ruff (0.13.1)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)